lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership_in_greece.govspeak.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :body do %>
+  ##Civil partnership in Greece
+
+  Contact the [Embassy of Greece](https://www.gov.uk/government/publications/foreign-embassies-in-the-uk) or the relevant authorities in Greece (town hall or [local notary public](http://www.notaries-directory.eu/) to find out about local marriage laws, including what documents you’ll need.  
+
+  ^You should get [legal advice](https://www.gov.uk/government/collections/list-of-lawyers) and check the [travel advice for Greece](https://www.gov.uk/foreign-travel-advice/greece) before making any plans.^
+
+  <% unless calculator.resident_of_uk? %>
+    You will need to provide proof of residence eg a residence permit, a utility statement or property ownership. 
+
+    ##What you need to do
+
+    The local [notary public](http://www.notaries-directory.eu/) in Greece will prepare a cohabitation agreement. You’ll need to sign it with your partner in front of the notary. 
+
+    You’ll also need to get the documentation [interpreted](https://www.gov.uk/government/publications/translators-and-interpreters-list) if you don’t speak Greek. 
+
+    You’ll both need your current passports or national identity cards. 
+
+    Once the agreement has been signed, you’ll need to register the agreement with the local town hall. The registrar will then issue you with a special registration certificate.
+  <% end %>
+
+  ##Naturalisation of your partner if they move to the UK
```

Comment:
There is a partial template (_partner_naturalisation_in_uk.govspeak.erb) where the wording is identical to what is written here. The only difference is that the partial links to the "Become a British citizen" page on GOV.UK.

There is a case to be made for reusing the partial here.